From d2ba3a8dc9e7564fcf20a2c5269d49af6c1aff40 Mon Sep 17 00:00:00 2001
From: Vicentiu Galanopulo <vicentiu@balena.io>
Date: Tue, 12 Oct 2021 14:09:44 +0200
Subject: [PATCH] Don't use gpio 119 as reset pin for GT911

This solves the leds issues on ep2.3, this pin is
connected to the led board OE on v2.3

Upstream-status: Inapropriate [configuration]
Signed-off-by: Alexandru Costache <alexandru@balena.io>
Signed-off-by: Vicentiu Galanopulo <vicentiu@balena.io>
---
 arch/arm64/boot/dts/compulab/cl-som-imx8.dts | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm64/boot/dts/compulab/cl-som-imx8.dts b/arch/arm64/boot/dts/compulab/cl-som-imx8.dts
index a5d039b13a41..153e221436ac 100644
--- a/arch/arm64/boot/dts/compulab/cl-som-imx8.dts
+++ b/arch/arm64/boot/dts/compulab/cl-som-imx8.dts
@@ -239,7 +239,6 @@
 		esd-recovery-timeout-ms = <2000>;
 		interrupts-extended = GPIRQ_GT911;
 		irq-gpios = GP_GT911_IRQ;
-		reset-gpios = GP_GT911_RESET;
 		status = "okay";
 	};
 
-- 
2.17.1

